spec/prog/vm/github_runner_spec.rb

```diff
@@ -502,12 +502,18 @@
 
   describe "#destroy" do
     it "naps if runner not deregistered yet" do
-      expect(client).to receive(:get)
+      expect(client).to receive(:get).and_return({busy: false})
```

Comment:
```suggestion
      expect(client).to receive(:get).and_return(busy: false)
```

I think this might work, but I'm ambivalent about this bit of syntactic simplification. ymmv.

Reply:
It worked. I don't have a strong preference either way. I just changed it.